### PR TITLE
docs(button): update usage page - INNO-2038

### PR DESCRIPTION
**Pull request by papegaill**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] `package.json` is up-to-date and `@ecl/[system]-base` is part of the dependencies
- [ ] I have checked the dependencies
- [ ] I have given the fractal status “ready” to my component
- [ ] I have declared `@define mycomponent` in the SCSS file
- [ ] I have specified `margin: 0;` on the CSS component
- [ ] I have provided tests
- [ ] I follow the naming guidelines
- [ ] the component supports composition
- [ ] there are no hardcoded strings (all content come from the context)
- [ ] I have filled the README.md file (at least a few lines)
- [ ] My component is listed in the root README
- [ ] My PR has the right label(s)
- [ ] The redirects in `src/website/src/routes/Redirects.jsx` are up-to-date

### CODE REVIEW

File: src/website/src/pages/ec/components/button/docs/usage.mdx [hunk 1]
```diff
@@ -3,70 +3,310 @@ title: Usage
...
+# Search Button
+
+<Paragraph size="lead">
+  The search button is only used with the search form.
+</Paragraph>
+
+## Anatomy
+
+<Anatomy
+  image="https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/button-search-anatomy-desktop.png"
```
Comment (emeryro) [inline src/website/src/pages/ec/components/button/docs/usage.mdx:251]: Images for mobile and desktop version are different here, so we should use an imgset (see https://github.com/ec-europa/europa-component-library/pull/1484/files#diff-af5dd2c1617b773729988b532636b636R21 for instance)

**Review by emeryro — CHANGES_REQUESTED** [review]
Just a small modification for the images, but apart of that this is good

File: src/website/src/pages/ec/components/button/docs/usage.mdx (hunk 1 above)
Comment (papegaill) [reply src/website/src/pages/ec/components/button/docs/usage.mdx:251]: Thank you for the review @emeryro, PR updated. That said, I don't quite understand the approach of using imgsrc .. so if you browse the style-guide on mobile you get mobile images .. doesn't make any sense.. even on desktop, I should be able to see mobile images and directive, what do you think?

**Review by kalinchernev — APPROVED** [review]

File: src/website/src/pages/ec/components/button/docs/usage.mdx (hunk 1 above)
Comment (kalinchernev) [reply src/website/src/pages/ec/components/button/docs/usage.mdx:251]: When we looked at the images on Friday with Alexis, there wasn't any difference at this moment between desktop and mobile. For me the PR is ok

Comment (emeryro) [reply src/website/src/pages/ec/components/button/docs/usage.mdx:251]: We would probably need a better way to display mobile/desktop version, I agree. Let's consider this as a temporary solution

**Review by emeryro — APPROVED** [review]

**Pull request merged**